specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -786,45 +800,55 @@
           }
         },
         "files": {
+          "description": "Files to be used in this session.",
```

Comment:
>session [](start = 51, length = 7)

I know that the description in livy doc used the word "session". But this property is one of SparkBatchJobRequest. So I think that we need to mitigate "session" to "spark batch job". How do you like?